[WIP] ROSA topic map restructure (#41923)
openshift/openshift-docs: rosa_getting_started/rosa-sts-interactive-mode-reference.adoc

:_content-type: ASSEMBLY
[id="rosa-sts-interactive-mode-reference"]
= Interactive cluster creation mode reference
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: rosa-sts-interactive-mode-reference

toc::[]

> This section provides an overview of the options that are presented when you use the interactive mode to create a cluster through the `rosa` CLI.

Reviewer: ```suggestion
You can customize many options when you use the interactive mode to create a cluster through the `rosa` CLI.
```
